### DISCUSSION TITLE
Health check for ECS Firelens in AWS Task definiation

### DISCUSSION THREAD

**Discussion by vijaynauti** [fluent/fluent-bit#6940] [Q&A]
Hi Team

I have started facing one issue with Firelens container. Where when task are failing its not sending logs to destination location.
For running task its working fine.

Also can someone please help to add the Firelens sidecar container with Healthcheck.

@agup006 @PettitWesley

**Comment by PettitWesley** [comment]
https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/health-check
